crates/bevy_log/src/lib.rs

```diff
@@ -51,15 +51,17 @@ use tracing_subscriber::{prelude::*, registry::Registry, EnvFilter};
 ///     App::new()
 ///         .insert_resource(LogSettings {
 ///             level: Level::DEBUG,
-///             filter: "wgpu=error,bevy_render=info".to_string(),
+///             filter: "wgpu=error,bevy_render=info,bevy_ecs::=trace".to_string(),
```

Comment:
I never saw this `bevy_ecs::=trace` syntax, how is it different from the `bevy_render=info` just before (aside from the level)?

Reply:
Did some more testing, and seems like there's no difference. Before I was under impression that ::= would include the submodules, but 'bevy_ecs=trace' does indeed print the same output. Maybe I should remove this bevy_ecs addition, and remove the :: from the BEVY_LOG example too?

Reply:
IMO just removing the `::` in both cases is better; showing how you can set the levels differently for different crates is interesting.

Reply:
Ok, I've removed the `::` now. Also added the `bevy_ecs=trace` to the `RUST_LOG` example